addons/l10n_se/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    "name": "Swedish - Accounting",
+    "version": "1.0",
+    "author": "XCLUDE",
+    "category": "Localization",
+    "depends": ["account", "base_vat"],
+    "data": [
+        "data/account_chart_template.xml",
+        "data/account.account.template.csv",
+        "data/account_chart_template_data.xml",
```

Comment:
Having account_chart_template, then account_chart_template_data seems weird (though I understand why you do that). Maybe just name them account_chart_template_before_accounts and account_chart_template_after_accounts.

Reply:
I renamed them according to your suggestion.